admin/modules/config/badwords.php

```diff
@@ -59,6 +59,7 @@
 	{
 		$new_badword = array(
 			"badword" => $db->escape_string($mybb->input['badword']),
+			"regex" => $mybb->get_input('regex', 1),
```

Comment:
Rather than using `1` here, can you use `MyBB::INPUT_INT` instead? it's a little bit more readable that way.